src/modules/site-v2/templates/about/people.html

```diff
@@ -1,83 +1,123 @@
-                {% endif %}
-
-                <!-- github -->
-                {% if p.github %}
-                    <img src="/static/img/icons/github.png" style="width:16px; "/> <a href="https://github.com/{{ p.github }}">{{ p.github }}</a><br />
-                {% endif %}
+{% block custom_head %}
+<style>
+    .card img {
+        width: 150px;
+        height: auto;
+    }
 
-                <br />
-                <p>{{ p.description }}</p>
+    .detailsLink {
+        font-size: 1.25rem !important;
+        color: #000;
+    }
+</style>
+{% endblock %}
 
-        	</div>
-        </div>
-    {% endfor %}
-    <br />
+{% block content %}
 
-    <!-- Committee -->
-    <h3>Scientific Advisory Committee</h3>
+<div class="container-fluid px-5 pb-5">
     <div class="row">
-    {% for p in profiles['committee']|sort(attribute='last_name') %}
-        <div class="col-md-12 clearfix">
-            
-            <img src="{{ p.img_url }}" style="width: 150px; " alt="{{ p.first_name }}" class="thumbnail hidden-xs lab-member-photo" />
-            <img src="{{ p.img_url }}" style="width: 150px; " alt="{{ p.first_name }}" class="thumbnail visible-xs lab-member-photo-small" />
-            
-            <br />
-            
-            <h4><a href="{{ p.website }}">{{ p.first_name }} {{ p.last_name }}</a></h4>
-            <h4><small>{{ p.org }}</small></h4>
-            
-            <br />
-
+        <div class="col col-md-3 mt-5">
+            <nav class="nav flex-column mx-auto" style="width:60%;">
+                <a class="nav-link" href="#staff">Staff</a>
+                <a class="nav-link" href="#SAC">Scientific Advisory Committee</a>
+                <a class="nav-link" href="#collaborators">Collaborators</a>
+            </nav>
         </div>
-    {% endfor %}
-    </div>
-    <br />
-
-    <!-- Collaborators -->
-    <h3>Collaborators</h3>
-    <div class="row">
-    {% for p in profiles['collaborators']|sort(attribute='last_name') %}
-        <div class="col-md-12 clearfix">
-            
-            <img src="{{ p.img_url }}" style="width: 150px; " alt="{{ p.first_name }}" class="thumbnail hidden-xs lab-member-photo" />
-            <img src="{{ p.img_url }}" style="width: 150px; " alt="{{ p.first_name }}" class="thumbnail visible-xs lab-member-photo-small" />
-            
-            <br />
-            
-            <h4><a href="{{ p.website }}">{{ p.first_name }} {{ p.last_name }}</a></h4>
-            <h4><small>{{ p.org }}</small></h4>
-            
-            <br />
-
+        <div class="col col-md-9 mt-5">
+            <!-- Staff section -->
+            <div class="row">
+                <h2 id="staff">Staff</h2>
+            </div>
+            <!-- /Row -->
+            <div class="row row-cols-1 row-cols-md-3 g-5 mb-5">
+                <!-- Card Column -->
+                {% for p in profiles['staff'] %}
```

Comment:
@vlagrassa - instead of `profiles['staff']` we should use a more defensive approach and have something like `profiles.get('staff', [])`. This will not break if the key is missing.